_includes/proposals.html

```diff
@@ -2,10 +2,10 @@
   <a id="proposals" tabindex="-1"></a>
   <h2 class="section-headline text-end">State of Proposals</h2>
   <div class="featurelist">
-    <h3>Stage 3</h3>
+    <h3>Upcoming additions to the Specification</h3>
```

Comment:
This header text is the part that I find most concerning—it seems that we're now implicitly promising that Stage 4 will be reached. We'd at least need the word "expected", but it's probably better to maintain the "candidate" verbiage.

Here's one possibility:
```suggestion
    <h3>Current Candidates for the Specification</h3>
```